File: CPP/Clothing.cpp
// library
#include "Produk.cpp"

// membuat kelas clothing yang merupakan turunan dari kelas produk
class Clothing: public Produk {

    public:
    string size;
    string material;
    string gender;
    
    public:
        // konstruktor kosong
        Clothing() {
        }

        // konstruktor dengan parameter
        Clothing(string size, string material, string gender) {
            // set nilainya
            this->size = size; 
            this->material = material;
            this->gender = gender;
        }

         // set & get ukuran
        void setSize(string size) {
            this->size = size;
        }

        string getSize() {
            return this->size;
        }

         // set & get material
        void setMaterial(string material) {
            this->material = material;
        }

        string getMaterial() {
            return this->material;
        }

         // set & get gender
        void setGender(string gender) {
            this->gender = gender;
        }

        string getGender() {
            return this->gender;
        }
        
        // deconstruct
        ~Clothing() {
        }
};

File: Python/Clothing.py
# membuat kelas clothing
class Clothing:
    # konstruktor dengan parameter
    def __init__(self, size, material, gender):
        self.size = size
        self.material = material
        self.gender = gender
    
    # tetapkan size
    def set_size(self, size):
        self.size = size
    
    # get size
    def get_size(self):
        return self.size
    
    # tetapkan material
    def set_material(self, material):
        self.material = material

    # get material
    def get_material(self):
        return self.material
    
    # tetapkan gender
    def set_gender(self, gender):
        self.gender = gender
    
    # get gender
    def get_gender(self):
        return self.gender